tests/performance/scale/tests/metadata-collector/main.go

```diff
@@ -276,6 +277,20 @@ func getGitRevision() (string, error) {
 	return hashString, nil
 }
 
+func truncateVersion(version string) string {
+    // Version should match x.y
+    re := regexp.MustCompile(`^\d+\.\d+`)
```

Comment:
I'm not sure if this regex is sufficient. In one example, we had this value: `v1.28.9+416ecaf`.
In that case, we would like to have `1.28` recorded in our metadata (at least that). Maybe we can use `^v?(\d+\.\d+)` - and extract the first group.

```
re := regexp.MustCompile(`^v?(\d+\.\d+)`)
match := re.FindStringSubmatch(version)
if len(match) > 0 {
  return match[1]
}
```

It would be even better to add a new field, `k8sFullVersion,` which can contain `clusterMetadata.K8SVersion`. In that case, we will have the proper type (string) in Elastic, and full versions will be recorded.

Reply:
Fixed.